bigchaindb/experimental/core.py

```diff
@@ -0,0 +1,26 @@
+from bigchaindb import App
```

Comment:
Moving the files around produces huge diffs, I would just give a real name to this folder from the start.